languages/javascript/maintainers.md

```diff
@@ -8,4 +8,4 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+- Georg Tavonius (Slack: @Georg Tavonius, GitHub: Calamari)
```

Comment:
```suggestion
- Georg Tavonius (Slack: @Georg Tavonius, GitHub: [Calamari](https://github.com/calamari))
```

Reply:
Maybe a link is nicer 😄

Reply:
Sure, I copied the style from another file, but does not mean we cannot improve it